.swiftlint.yml

```diff
@@ -29,6 +29,8 @@ disabled_rules:
   - xctfail_message
   - empty_count
   - overridden_super_call
+  - redundant_string_enum_value
```

Comment:
Disabled because it can't be autofixed and it's a minor violation and I want to avoid manual changes when possible